feed_api/app.py

```diff
@@ -29,6 +38,22 @@
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
+API_KEY_NAME = "bsky-internal-api-key"
+REQUIRED_API_KEY = json.loads(
```

Comment:
Note: this should be ideally cached, otherwise every time that this API is pinged, the load happens.

I wonder if lambda would cache this?

Reply:
LGTM! Great job team Mark!